src/System.Runtime/tests/System/GCTests.netstandard.cs

```diff
@@ -90,35 +90,103 @@ public static void GCNotificationTests(bool approach, int timeout)
 
         [Fact]
         [OuterLoop]
-        public static void TryStartNoGCRegionNegTest()
+        public static void TryStartNoGCRegionNegTest1()
```

Comment:
Nit: rather than "Test1", "Test2", "Test3", etc., could you give these more descriptive names that describe what's unique about the test? e.g. "TryStartNoGCRegionNegTest1" => "EndNoGCRegion_NoMatchingStart_ThrowsInvalidOperationException"